README.md

```diff
@@ -8,43 +8,147 @@ Humans tend to build new models around existing ones, which can be counterproduc
+
+### Copernican Heliocentrism
+In the 1500’s Copernicus observed the complex motion of the planets in the sky and postulated a heliocentric model as the simplest representation of the solar system. To investigate the same process, we posed the problem to SciNet of predicting, based on prior positions, the angles of the Sun and Mars as seen from Earth, at a future time. That is, describing the time evolution of the planetary orbits.
+
+![helio](https://github.com/nmdickso/Phys490FinalProject/blob/veronica/images/helio.JPG)
+
+In order to simulate time evolution, a small feed-forward network is introduced after the representation, before the decoder is applied, transforming SciNet into a recurrent neural network. In the paper, it is shown that the information stored in the time-evolved latent representation corresponds to the angles of the planets as seen from the Sun, demonstrating that the simplest representation it can find is, as Copernicus discovered, the heliocentric model.
+
+### Representation of Qubits
+An interesting property of SciNet is its ability to determine properties of physical systems with no prior theoretical structure. In the case of determining the dimensionality of a Hilbert space, SciNet is given the average "measurement identical states" psi, with respect to "basis states" phi. The following question is then posed: “What is the result of a measurement of psi with respect to a random state omega?”
+
+![Qubits](https://github.com/nmdickso/Phys490FinalProject/blob/veronica/images/qubits.JPG)
+
+Omega is parameterized in such a way as to not hint at any human invented structure. For tomographically complete observations, SciNet’s error drops with the number of representation neurons until it equals the degrees of freedom of the Hilbert Space, after which the predictions have near perfect accuracy. 
+
+For tomographically *incomplete* data, SciNet's error plateaus before the number of representation neurons reaches the degrees of freedom of the Hilbert Space, with further representation neurons not decreasing in error.
+
+#### Extending the Representation of Qubits Problem
+To extend this problem, we can pose the question: "What would happen if psi consisted of an entangled 2 qubit system and a non-fixed input representing the average result of the partial measurement of the first system? Omega would be the parameterization of a partial measurement on the second system. Would SciNet respond in a similar manner to decreasing the number of latent neurons for both non-entangled and entangled partial measurements? 
+
+# Technical Instructions
+
+## Requirements
 
 - Python 3.x
 - ``numpy``
 - ``matplotlib``
+- ``mpl_toolkits``
+- ``setuptools``
+- ``itertools``
+- ``argparse``
+- ``datetime``
+- ``random``
```

Comment:
Packages which are either standard library or a dependency of another package should not be listed as requirements.

Reply:
I can change that real quick